sklego/common.py

```diff
@@ -0,0 +1,85 @@
+import hashlib
+
+import numpy as np
+import pandas as pd
+from sklearn.utils.validation import check_is_fitted, check_array, check_X_y
+
+
+class TransformerSelectorMixin:
```

Comment:
so .... `TrainOnlyTransformerMixin` ?

Reply:
I've also thought about that one and it does sound better but in theory you can use it to transform train and test differently, not just only transform train.
I do agree that transforming train only is probably going to be the main use case so I might as well call it that.